### PR TITLE
Reference-data-from-annotation-table

### PR DESCRIPTION
**Pull request by J-81**
Updated code to pull reference genome metadata from annotations csv file.
Removed deprecated and unused RNAseq/organisms.csv

### CODE REVIEW

File: RNAseq/Pipeline_GL-DPPD-7101_Versions/GL-DPPD-7101-F.md [hunk 1]
```diff
@@ -39,6 +39,7 @@ The DESeq2 Normalization and DGE step, [step 9](#9-normalize-read-counts-perform
 
 # Table of contents  
 
+
```
Comment (J-81) [inline RNAseq/Pipeline_GL-DPPD-7101_Versions/GL-DPPD-7101-F.md:42]: test comment: this should be removed

Comment (J-81) [reply RNAseq/Pipeline_GL-DPPD-7101_Versions/GL-DPPD-7101-F.md]: And I did go ahead and remove it

File: RNAseq/Workflow_Documentation/NF_RCP-F/README.md [hunk 2]
```diff
@@ -67,7 +68,7 @@ document](../../Pipeline_GL-DPPD-7101_Versions/GL-DPPD-7101-F.md):
 
 ---
 
-### 1. Install Nextflow and Singularity 
+### 1. Install Nextflow and Singularity
```
Comment (J-81) [inline RNAseq/Workflow_Documentation/NF_RCP-F/README.md:71]: Removing extra whitespace is a good idea here, keeping this in the PR

File: RNAseq/Workflow_Documentation/NF_RCP-F/README.md [hunk 3]
```diff
@@ -1,12 +1,12 @@
...
+The current GeneLab RNAseq consensus processing pipeline (RCP), [GL-DPPD-7101-F](../../Pipeline_GL-DPPD-7101_Versions/GL-DPPD-7101-F.md), is implemented as a [Nextflow](https://nextflow.io/) DSL2 workflow and utilizes [Singularity](https://docs.sylabs.io/guides/3.10/user-guide/introduction.html) to run all tools in containers. This workflow (NF_RCP-F) is run using the command line interface (CLI) of any unix-based system.  While knowledge of creating workflows in Nextflow is not required to run the workflow as is, [the Nextflow documentation](https://nextflow.io/docs/latest/index.html) is a useful resource for users who want to modify and/or extend this workflow.
```
Comment (J-81) [inline RNAseq/Workflow_Documentation/NF_RCP-F/README.md:7]: Removing extra whitespace is a good idea here, keeping this in the PR

File: RNAseq/Workflow_Documentation/NF_RCP-F/README.md [hunk 4]
```diff
@@ -1,12 +1,12 @@
-# NF_RCP-F Workflow Information and Usage Instructions
+# NF_RCP-F Workflow Information and Usage Instructions <!-- omit in toc -->
```
Comment (J-81) [inline RNAseq/Workflow_Documentation/NF_RCP-F/README.md:1]: Should be kept, this prevent vscode markdown toc generation from including these headers.

File: RNAseq/Workflow_Documentation/NF_RCP-F/README.md [hunk 5]
```diff
@@ -1,12 +1,12 @@
...
 
-## General Workflow Info
+## General Workflow Info <!-- omit in toc -->
 
-### Implementation Tools
+### Implementation Tools <!-- omit in toc -->
```
Comment (J-81) [inline RNAseq/Workflow_Documentation/NF_RCP-F/README.md:5]: Should be kept, this prevent vscode markdown toc generation from including these headers.

File: RNAseq/Workflow_Documentation/NF_RCP-F/README.md [hunk 6]
```diff
@@ -1,12 +1,12 @@
...
+The current GeneLab RNAseq consensus processing pipeline (RCP), [GL-DPPD-7101-F](../../Pipeline_GL-DPPD-7101_Versions/GL-DPPD-7101-F.md), is implemented as a [Nextflow](https://nextflow.io/) DSL2 workflow and utilizes [Singularity](https://docs.sylabs.io/guides/3.10/user-guide/introduction.html) to run all tools in containers. This workflow (NF_RCP-F) is run using the command line interface (CLI) of any unix-based system.  While knowledge of creating workflows in Nextflow is not required to run the workflow as is, [the Nextflow documentation](https://nextflow.io/docs/latest/index.html) is a useful resource for users who want to modify and/or extend this workflow.
 
-### Workflow & Subworkflows
+### Workflow & Subworkflows <!-- omit in toc -->
```
Comment (J-81) [inline RNAseq/Workflow_Documentation/NF_RCP-F/README.md:9]: Should be kept, this prevent vscode markdown toc generation from including these headers.

File: RNAseq/organisms.csv [hunk 7]
```diff
@@ -1,11 +0,0 @@
-name,species,taxon,annotations
-HUMAN,Homo sapiens,9606,org.Hs.eg.db
```
Comment (J-81) [inline RNAseq/organisms.csv:2]: Removed as the updated workflow replaces this file with the annotation tables

Comment (J-81) [reply RNAseq/organisms.csv:2]: I'll be relinking any older DPPD documents to a Github commit url.

File: RNAseq/Workflow_Documentation/NF_RCP-F/README.md (hunk 3 above)
Comment (J-81) [reply RNAseq/Workflow_Documentation/NF_RCP-F/README.md:7]: This can actually affect formatting in certain cases, better to leave with spaces.

**Review by asaravia-butler — APPROVED** [review]

**Pull request merged**